diff --git a/dist/cliHandlers.js b/dist/cliHandlers.js
index 8e619b6178e76692cbde719ca1224020fb4ad4b8..84178535c8e3d30e03425bd9f217d0859d17fb5e 100644
--- a/dist/cliHandlers.js
+++ b/dist/cliHandlers.js
@@ -85,7 +85,7 @@ const apiServerHandler = async options => {
     fastify
   }).ready(() => {
     console.log(_ansiColors.default.italic(_ansiColors.default.dim('Took ' + ((0, _now.default)() - tsApiServer) + ' ms')));
-    const on = socket ? socket : _ansiColors.default.magenta(`http://localhost:${port}${apiRootPath}`);
+    const on = socket ? socket : _ansiColors.default.magenta(`http://0.0.0.0:${port}${apiRootPath}`);
     console.log(`API listening on ${on}`);
     const graphqlEnd = _ansiColors.default.magenta(`${apiRootPath}graphql`);
     console.log(`GraphQL endpoint at ${graphqlEnd}`);
diff --git a/dist/server.js b/dist/server.js
index 31719a264f277bb1c7b09fcb5174dcfd5936602e..777122a9366188d499f32a6e1e5d0b0e112ee867 100644
--- a/dist/server.js
+++ b/dist/server.js
@@ -12,7 +12,7 @@ const startServer = ({
   socket,
   fastify
 }) => {
-  const host = process.env.NODE_ENV === 'production' ? '0.0.0.0' : '::';
+  const host = '0.0.0.0';
   const serverPort = socket ? (0, _parseInt2.default)(socket) : port;
   fastify.listen({
     port: serverPort,
